### PR TITLE
move the refresh to live in parseOptions to make sure it works after hydration

### PR DESCRIPTION
**Pull request by saponifi3d**
basically what the title says.

it's a quick update of: https://github.com/rendrjs/rendr/pull/304 - and adds tests around it.

### CODE REVIEW

**Comment by mattpardee** [comment]
Did not even know this option existed. LGTM!

**Comment by quaelin** [comment]
:+1:

File: test/shared/base/view.test.js [hunk 1]
```diff
@@ -200,6 +202,23 @@ describe('BaseView', function() {
       view.parseOptions(options);
       view.options.should.deep.equal(options);
     });
+
+    it('adds a refresh listener if rendrOnRefresh is set', function () {
```
Comment (spikebrehm) [inline test/shared/base/view.test.js]: Small typo -- `rendrOnRefresh`.  That muscle memory :)

**Comment by spikebrehm** [comment]
thanks :+1:

**Pull request merged**